src/main/java/Bob.java

```diff
@@ -0,0 +1,338 @@
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.time.LocalDate;
+import java.time.format.DateTimeFormatter;
+import java.time.format.DateTimeParseException;
+import java.util.Arrays;
+import java.util.Scanner;
+import java.util.ArrayList;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Paths;
+import java.nio.file.Path;
+import org.apache.commons.lang3.StringUtils;
+
+public class Bob {
```

Comment:
Should split into more classes (UI, Storage, TaskList)

Reply:
Yup, that will be under A-MoreOOP which I believe Leonard has not committed. Jiayou!